jumanji/training/configs/env/game_2048.yaml

```diff
@@ -1,5 +1,5 @@
 name: game_2048
-registered_version: Game2048-v0
+registered_version: Game2048-v1
```

Comment:
Remind me why we are incrementing this - was it because of the bug that was found?

Reply:
It was already incremented, this fixes the training configs. It was incremented due to a bug in the action mask in #144

Reply:
Ok 👍